docs/mkdocs.yml

```
@@ -0,0 +1,94 @@
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
+
+site_name: Event Sourcing
+site_url: https://patchlevel.de/
+repo_url: https://github.com/patchlevel/event-sourcing-bundle
+repo_name: patchlevel/event-sourcing-bundle
+edit_uri: edit/2.0.x/docs/pages/
+docs_dir: pages
+
+extra:
+  meta:
+    title: Event Sourcing Bundle
+    description: A lightweight but also all-inclusive event sourcing library with a focus on developer experience.
```

```suggestion
    description: A lightweight but also all-inclusive event sourcing bundle with a focus on developer experience and based on doctrine dbal.
```

this is what we have right now here in github.